typescript/packages/core/src/types/Chain.ts

```diff
@@ -12,7 +12,8 @@ export type Chain =
     | ZilliqaChain
     | CosmosChain
     | StarknetChain
-    | RadixChain;
+    | RadixChain
+    | SeiChain;
```

Comment:
Sei is EVM compatible so no need to do this :)

Reply:
It's also not even used in the plugin

Reply:
ah, I added it because I thought Sei is a new chain